src/nRF51822.cpp

```diff
@@ -1,12 +1,21 @@
 // Copyright (c) Sandeep Mistry. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-#if defined(NRF51) || defined(NRF52) || defined(__RFduino__)
+#ifdef __Simblee__ // TODO: Is this actually correct? It's the only way this file compiles for me using the S110 V8 headers.
+#define NRF5
```

Comment:
Not sure I'd recommend this, better to add the equivalents in future down in the file. In case the nRF5 core updates the S110 version.